src/main/java/com/stormnet/page/ReviewPage.java

```diff
@@ -0,0 +1,33 @@
+package com.stormnet.page;
+
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ReviewPage {
+  private SelenideElement starsMark = $("div.star>a[title=\"5\"]");
+  private SelenideElement commentTitle = $("#comment_title");
+  private SelenideElement reviewContent = $("#content");
+  private SelenideElement submitReview = $("#submitNewMessage");
+
+  public ReviewPage setStarMark() {
+    starsMark.hover().click();
+    return this;
+  }
+
+  public ReviewPage setCommentTitle() {
+    commentTitle.sendKeys("High quality product");
+    return this;
+  }
+
+  public ReviewPage setReviewContent() {
+    reviewContent.sendKeys(
```

Comment:
Why always send this message, what if I want to send another message?

Reply:
fixed